transitions/generate_patient_id_on_patients.js

```diff
@@ -0,0 +1,11 @@
+var transitionUtils = require('./utils');
+
+module.exports = {
+  filter: function(doc) {
+    return doc.type === 'person' &&
```

Comment:
Doesn't this mean all `person` type docs will get a patient_id?  I'm not sure if that's the intent because I don't have enough context but that's pretty odd behavior to me.  Should there be a way to enable/disable which person docs have patient ids?